File: 100-Days-Series/Day 80/MultiplicationTable.java
class MultiplicationTable{
    // Method to print multiplication table for a given number
    public static void getPrintTable(int number){
        System.out.println("Multiplication table for "+number+" :");
        for(int i=1;i<=10;i++){
            //Correcting the multiplication logic in the loop
            System.out.println(number+" x "+i+" = "+(number*i));
        }
    }
    // Main Method
    public static void main(String [] args){
        // Let's Start Here
        int number;
        java.util.Scanner sc=new java.util.Scanner(System.in);
        // Prompt the user to enter a number
        System.out.print("Enter a number to print its multiplication table : ");
        number=sc.nextInt();
        // call the function to print the multiplication table
        getPrintTable(number);
    }
}

File: 100-Days-Series/Day 80/MultiplicationTable.py

# Function to print the multiplication table for a given number.
def get_print_table(number):
    print("Multiplication table for",number," :")
    for i in range(1,11,1):
        print(number," x ",i," = ",(number*i))


# Let's Start here
# Prompt the user to enter a number
number=int(input('Enter a number to print its multiplication table : '))
# Call the function to print its multiplication table
get_print_table(number)